src/Java.Interop/Java.Interop/JniValueMarshalerAttribute.cs

```diff
@@ -4,7 +4,7 @@
 namespace Java.Interop {
 
 	[AttributeUsage (Targets, AllowMultiple=false)]
-	public class JniValueMarshalerAttribute : Attribute {
+	public sealed class JniValueMarshalerAttribute : Attribute {
```

Comment:
It should be noted that this is an API break -- ...though is anybody actually using it?  *Could* anyone actually use it?  ¯\_(ツ)_/¯ -- but, "more" importantly, it contravenes FxDG suggestions:

[**X DO NOT seal classes without having a good reason to do so.**](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/sealing)

The "obvious" rationale for *not* sealing this type is so that subclasses can provide their own default constructor:

```cs
public partial class MyColorMarshalerAttribute : JniValueMarshalerAttribute {
    public MyColorMarshaler : base (typeof (MyColorMarshaler)) {
    }
}
```

I don't see why we'd want to lose this ability.

Reply:
I have no preference here (and it is a minor breaking change), but if the deciding factor is the FxDG, it suggests sealing attributes (which is why there's a rule for it):

[✓ DO seal custom attribute classes, if possible. This makes the look-up for the attribute faster](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/attributes)

I can add a suppression if we'd rather not make this change.

Reply:
I think i'd rather have a suppression.

Reply:
Sounds good, done!